README.md

```diff
@@ -47,7 +47,7 @@ For secrets in the same AWS account as the agent, you can use the secret name ra
 steps:
   - commands: 'echo \$MY_SECRET'
     plugins:
-      - seek-oss/aws-sm#v2.3.2:
```

Comment:
The Buildkite plugin linter checks the README to ensure that examples point at the right repo! This was a surprise.

Reply:
Kinda neat.